src/vivarium_csu_hypertension_sdc/model_specifications/test_ihd.yaml

```diff
@@ -12,6 +12,17 @@ components:
 
     vivarium_csu_hypertension_sdc.components:
         - IschemicHeartDisease()
+        - Stroke('ischemic_stroke')
+        - Stroke('intracerebral_hemorrhage')
+        - Stroke('subarachnoid_hemorrhage')
+        - BetterRiskEffect('risk_factor.high_systolic_blood_pressure', 'sequela.acute_myocardial_infarction.incidence_rate')
```

Comment:
I do really like how explicit these risk effects are to make it clear there are two rates in each model affected